docs/utils/atom-with-storage.mdx

```diff
@@ -68,6 +68,12 @@ const TextBox = () => {
     <>
       <input value={text} onChange={(e) => setText(e.target.value)} />
       <button onClick={() => setText(RESET)}>Reset (to 'hello')</button>
+      <button
+        onClick={(e) =>
+          setText((prev) => (prev === 'reset' ? RESET : e.target.value))
```

Comment:
Hmm? I don't think this example is very convincing.

Reply:
Should I add an explanation? or a separate example?

Reply:
Ah, I was using the wrong. HTML element type. Adjusted here, let me know if this is an appropriate example

[ca565fb](https://github.com/pmndrs/jotai/pull/1346/commits/ca565fb01414b4e030c82bb9f04f37b940cbe4c8)

Reply:
I mean, when does `prev` become `'reset'`? My interpretation is you type "reset" and type another key, it will be reset. It's weird or not realistic usage.
If you want to type "reset" to make it reset, this should work:
```js
setText(e.target.value === 'prev' ? RESET : e.target.value)
```
It's still not realistic, though.

Reply:
Agreed. Its a weird example, I've added a new example that might be more appropriate here: https://github.com/pmndrs/jotai/pull/1346/commits/3059bcfc8fcba3469c7e3d6f614d4f825ab38f27